ftw/iframeblock/browser/iframeblock.py

```diff
@@ -1,12 +1,15 @@
 from ftw.simplelayout.browser.blocks.base import BaseBlock
 from plone import api
-from textwrap import dedent
 from urlparse import urlparse
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+import json
 
 
 class IFrameBlockView(BaseBlock):
+    template = ViewPageTemplateFile('templates/iframeblock.pt')
+
     def __call__(self):
-        return self.index()
+        return self.template()
```

Comment:
This is the same as the superclass.... so redundant, right?

Reply:
Forgot that `SimeplLayoutBlockView` already calls the template. sry

Reply:
Ok, so we can remove this :-) Then  it's "go go go" from me :-)